src/TropicalGeometry/TropicalPolyhedron/constructors.jl

```diff
@@ -0,0 +1,151 @@
+const MinOrMax = Union{typeof(min), typeof(max)}
+
+struct TropicalPolyhedron{M<:MinOrMax} <: PolyhedralObject{QQFieldElem}
+  pm_tpolytope::Polymake.BigObject
+
+  @doc raw"""
+      TropicalPolyhedron{M}(p::Polymake.BigObject) where M<:MinOrMax
+
+  Constructs a `TropicalPolyhedron` corresponding to a `Polymake.BigObject` with tropical addition `M`.
```

Comment:
```suggestion
  Construct a `TropicalPolyhedron` corresponding to a `Polymake.BigObject` with tropical addition `M`.
```

Consistency with other parts of OSCAR code.